test/gtest/uct/ib/test_dc.cc

```diff
@@ -532,6 +520,19 @@ UCS_TEST_P(test_dc_flow_control, pending_grant)
     flush();
 }
 
+UCS_TEST_P(test_dc_flow_control, pending_grant_and_resend_hard_req,
+           "DC_FC_HARD_REQ_TIMEOUT=0.1s")
+{
+    /* test uses manipulation with available TX resources which may break
+       DCI allocation ordering. allow out-of-order DCI waitq */
+    ignore_dci_waitq_reorder(m_e2);
```

Comment:
maybe extend test_pending_grant in test_dc class , so in addition it will call ignore_dci_waitq_reorder before test_rc::test_pending_grant?

Reply:
done